test/extended/operators/olm.go

```diff
@@ -208,6 +209,11 @@ var _ = g.Describe("[sig-operator] an end user can use OLM", func() {
 			g.Skip("redhat-operators source not found in enabled sources")
 		}
 
+		// TODO: Come up with an arch agnostic operator to deploy during this test.
+		// For now this does not work for arm64
+		if e2e.NodeOSDistroIs("arm64") {
```

Comment:
is `NodeOSDistroIs` used for identifying the OS type rather than arch? is there an equivalent for architecture?

Reply:
It looks like `NodeOSArchIs` may be more applicable here:

```go
// NodeOSArchIs returns true if the node OS arch is included in the supportedNodeOsArchs. Otherwise false.
func NodeOSArchIs(supportedNodeOsArchs ...string) bool {
	for _, arch := range supportedNodeOsArchs {
		if strings.EqualFold(arch, TestContext.NodeOSArch) {
			return true
		}
	}
	return false
}
```

There also looks like there's a `SkipUnlessNodeOSArchIs(...)` that will skip tests unless the list of arch(es) provided as arguments match the test context.

Reply:
Oops. I meant to use `NodeOSArchIs`. Thanks for catching it @Prashanth684!

Reply:
@timflannagan yea unfortunately doesn't look like there's a `SkipIfNodeOSArchIs` :( and hard coding the list of archs we want the tests to run on doesn't seem like a good idea. So just used `NodeOSArchIs`.